docs/source/upcoming_release_notes/1123-square-one_tri-sphere_motion_system.rst

```diff
@@ -0,0 +1,30 @@
+1123 square-one tri-sphere motion system
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- SQR1
```

Comment:
Can you please include context information here? Even if it's just the header of your module docstring?

At this point in the review I'm kind of just saying "read and follow the pre-merge checklist" repeatedly which is probably unpleasant for both of us.